content/en/docs/getting-started-guides/windows/_index.md

```diff
@@ -151,6 +151,12 @@ Note: this file assumes that a user previous created 'l2bridge' host networks on
 
 #### For 3. Open vSwitch (OVS) & Open Virtual Network (OVN) with Overlay
 
+{{< note >}}
+**Note:** Fully automated setup via Ansible playbooks is available [here](https://github.com/openvswitch/ovn-kubernetes/tree/master/contrib)
+{{< /note >}}
+
+For manual setup, please continue reading the following steps.
```

Comment:
No need for "please" - see https://kubernetes.io/docs/home/contribute/style-guide/#content-best-practices